SparkPost/2web2ui
# FE-1224 - Upgrade Hibana Matchbox to version 4.4.0 and add relevant component wrappers

[FE-1224](https://sparkpost.atlassian.net/browse/FE-1224)

### What Changed
- Update Matchbox and related dependencies to `4.4.0`
- Adds component wrappers for the `Picture` and `Video` components
- Removed our custom implementation of `Picture`

### How To Test
- On `/dashboardV2`, verify the image still renders as expected
- Verify nothing breaks - especially empty states as we've seen on other PRs. Best way to do that is to sign up for a new account and click on several features to make sure things are rendering (though Cypress should catch this)

### To Do
